### DISCUSSION TITLE
Snippet's Filename attribute uses absolute path. Would relative path make more sense?

### DISCUSSION THREAD

**Discussion by csessh** [knqyf263/pet#317] [General]
Let's say I have this config:
 
```
[General]
Backend = "gist"
Cmd = []
Color = true
Column = 50
Editor = "nvim"
Format = "[$tags] [$description] > $command"
SelectCmd = "fzf --ansi --highlight-line --layout=reverse --border --height=90% --pointer=* --cycle --prompt=Snippets:"
SnippetDirs = []
SnippetFile = "~/.config/pet/snippet.toml"
SortBy = "recency"
```

I specify a snippet.toml file path like so:
```
SnippetFile = "~/.config/pet/snippet.toml"
```

`pet new` would create an entry like this: 
```
[[Snippets]]
  Description = "Prune git repo"
  Filename = "/home/csessh/.config/pet/snippet.toml"
  Output = ""
  Tag = ["git"]
  command = "git fetch --all --prune"
```

Filename attribute is displayed as absolute path. This doesn't play too well with porting configs/snippets from one workstation to another if they don't share the same username.

Is there a way to enforce snippet entries to keep the relative format? 

or is there a particular reason for them to be saved in absolute path format? If there isn't, I'd be happy to submit a PR.

**Comment by csessh** [comment]
Putting snippet file under `/tmp/*` works around this issue just fine. 
I suppose it will do for me for now

**Comment by RamiAwar** [comment]
Mmm it should be possible, but I don't think I've touched this part of the code before. I'd be interested in adding this functionality though, your concern makes sense!

**Reply by csessh** [reply]
I'm happy to take this and submit a PR later. 
Feel free to reopen this issue if necessary

**Reply by RamiAwar** [reply]
Yes please! Actually created an issue out of this.
